src/main/java/com/epam/aidial/cfg/domain/mapper/KeyCoreMapper.java

```diff
@@ -28,6 +30,17 @@ public interface KeyCoreMapper {
     @Mapping(target = "name", source = "name")
     Key mapKey(CoreKey key, String name);
 
+    @Mapping(target = "projectContactPoint", ignore = true)
```

Comment:
i'd use new method in old Key mapKey(CoreKey key, String name) with newObj. it helps to avoid @Mapping annotation duplication